content/docs/command-reference/remove.md

```diff
@@ -62,12 +61,9 @@ Remove `data.csv` data file:
 $ dvc remove data.csv.dvc
 $ ls data.csv*
 
-     data.csv.dvc
-```
+     data.csv
 
-Purge `.dvc` files:
+$ cat .gitignore | wc -l
 
```

Comment:
```suggestion
$ cat .gitignore | wc -l
```

Reply:
I'm not sure about this one. It's a little complex, is there a more obvious way to check that .gitignore is empty that doesn't assume people know GNU commands well?

Reply:
I have changed the example. Not sure it became better though. Please, check it out.

Reply:
It's simpler for sure, that's good. Thanks!